lib/mobile_robot_simulator/robot_models/__init__.py

```diff
@@ -16,10 +16,13 @@
 # You should have received a copy of the GNU General Public License
 # along with this program. If not, see <http://www.gnu.org/licenses/>.
 
-import robot_models
-import vectorfield
-def main():
-    pass
+from .differential_robot import Differential_robot
```

Comment:
add `from __future__ import absolute_import` before this relative imports, please